julia.spdx.json

```diff
@@ -0,0 +1,508 @@
+        },
+        {
+            "name": "OpenBLAS",
+            "SPDXID": "SPDXRef-OpenBLAS",
+            "downloadLocation": "git+https://github.com/xianyi/OpenBLAS.git",
+            "filesAnalyzed": false,
+            "homepage": "https://www.openblas.net",
+            "sourceInfo": "The git hash of the version in use can be found in the file deps/openblas.version",
+            "licenseConcluded": "BSD-3-Clause",
+            "licenseDeclared": "BSD-3-Clause",
+            "copyrightText": "Copyright (c) 2011-2014, The OpenBLAS Project",
+            "summary": "OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version."
+        },
+        {
+            "name": "LAPACK",
+            "SPDXID": "SPDXRef-LAPACK",
+            "downloadLocation": "https://www.netlib.org/lapack/",
+            "filesAnalyzed": false,
+            "homepage": "https://netlib.org/",
+            "sourceInfo": "The version in use can be found in the file deps/Versions.make",
+            "licenseConcluded": "BSD-3-Clause",
+            "licenseDeclared": "BSD-3-Clause",
+            "copyrightText": "Copyright (c) 1992-2013 The University of Tennessee and The University of Tennessee Research Foundation.  All rights reserved.\nCopyright (c) 2000-2013 The University of California Berkeley. All rights reserved.\nCopyright (c) 2006-2013 The University of Colorado Denver.  All rights reserved.",
+            "summary": "LAPACK is written in Fortran 90 and provides routines for solving systems of simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue problems, and singular value problems."
+        },
+        {
+            "name": "PCRE",
+            "SPDXID": "SPDXRef-PCRE",
+            "downloadLocation": "https://ftp.pcre.org/pub/pcre/",
+            "filesAnalyzed": false,
+            "homepage": "https://www.pcre.org",
+            "sourceInfo": "The version in use can be found in the file deps/Versions.make",
+            "licenseConcluded": "BSD-3-Clause",
+            "licenseDeclared": "BSD-3-Clause",
+            "copyrightText": "Copyright (c) 1997-2021 University of Cambridge All rights reserved.\nCopyright(c) 2009-2021 Zoltan Herczeg\n",
+            "summary": "PCRE2 is a library of functions to support regular expressions whose syntax and semantics are as close as possible to those of the Perl 5 language."
+        },
+        {
+            "name": "LibSuiteSparse",
+            "SPDXID": "SPDXRef-LibSuiteSparse",
+            "downloadLocation": "git+https://github.com/DrTimothyAldenDavis/SuiteSparse",
+            "filesAnalyzed": false,
+            "homepage": "https://people.engr.tamu.edu/davis/suitesparse.html",
+            "sourceInfo": "The version in use can be found in the file deps/Versions.make",
+            "licenseConcluded": "NOASSERTION",
```

Comment:
Since each module of SuiteSparse has its own unique license, I found it made sense to list the modules individually in the SPDX module.

I think I'll go back later and add an issue about how the default build options result in a GPL licensed binary.